src/views/activity/components/transaction-card/transaction-card.view.tsx

```diff
@@ -54,43 +43,39 @@ const TransactionCard: FC<TransactionCardProps> = ({
         </div>
         <div className={classes.actionColumn}>
           <Typography type="body1">{actionText}</Typography>
-          {status === "completed" ? (
-            <Typography type="body2" className={classes.time}>
-              {getTimeFromNow({ timestamp })}
-            </Typography>
-          ) : (
-            <span
-              className={`${classes.statusBox} ${
-                status === "on-hold" || status === "failed" ? classes.redStatus : ""
-              }`}
-            >
-              {getTransactionStatusText(status)}
-            </span>
-          )}
+          <span
+            className={`${classes.statusBox} ${status === "completed" ? classes.greenStatus : ""}`}
+          >
+            {getTransactionStatusText(status)}
+          </span>
         </div>
         <div className={classes.tokenColumn}>
           <div className={classes.token}>
-            {env && <Icon url={env.tokens.ETH.logoURI} className={classes.tokenIcon} size={20} />}
-            <Typography type="body1">
-              {amount} {token.toUpperCase()}
-            </Typography>
+            <Icon url={transaction.token.logoURI} className={classes.tokenIcon} size={20} />
+            <Typography type="body1">{formatEther(amount)} ETH</Typography>
```

Comment:
Don't use `formatEther`, be more generic and do: `${formatUnits(amount, token.decimals)} ${token.symbol}`. This will support ERC-20 tokens in the future.

Reply:
Right. Done!